kernel_gateway/gatewayapp.py

```diff
@@ -312,19 +320,23 @@ def init_configurables(self):
                     self.prespawn_count, self.max_kernels)
                 )
 
-        if self.api == 'notebook-http':
-            self.kernel_pool = ManagedKernelPool(
-                self.prespawn_count,
-                self.kernel_manager
-            )
-        else:
-            self.kernel_pool = KernelPool(
-                self.prespawn_count,
-                self.kernel_manager
-            )
+        
+        api_module = self._load_api_module(self.api)
+        func = getattr(api_module, 'create_personality', None)
```

Comment:
I'm a little confused about the second arg, the `None`. When is this not None? Why pass it in if it's always None?